actian_jdbc/connectionBuilder.js

```diff
@@ -1,29 +1,6 @@
 (function dsbuilder(attr)
 {
-    var params = [];
-
-    if (attr["server"].toLowerCase() == "(local)")  // match ODBC implementation, even though JDBC driver does not understand (local)
-    {
-        attr["server"] = "localhost";
-        // local connection, no auth specified
-        // port is required
-    }
-    else
-    {
-        params["UID"] = attr["username"];
-        params["PWD"] = attr["password"];
-    }
-
-    var urlBuilder = "jdbc:ingres://" + attr["server"] + ":" + attr["port"] + "/" + attr["dbname"] + ";";
-
-    var formattedParams = [];
-
-    for (var key in params) {
-        formattedParams.push(connectionHelper.formatKeyValuePair(key, params[key]));
-    }
-
-    urlBuilder += formattedParams.join(";")
+    var urlBuilder = "jdbc:ingres://" + attr[connectionHelper.attributeServer] + ":" + attr[connectionHelper.attributePort] + "/" + attr[connectionHelper.attributeDatabase] + ";";
```

Comment:
I'd like to preserve the `(localhost)` behavior is possible for easy of copy/paste and testing the different connectors

Reply:
I'd like to be sure and understand the request. Without specifically adding any code in the Actian tacos for handling `(localhost)`, the current behavior works fine with both the JDBC and ODBC tacos when using `localhost` (without the parenthesis) for the **Server** field.

![image](https://user-images.githubusercontent.com/115743596/208961519-4b8953a6-7715-4793-8a6a-e7adf92b0e26.png)

Do we need to add support for a Server value of `(localhost)`?

Reply:
locahost under ODBC will require auth, (localhost) does not. I would like to see support for both so that copy/paste config works for both,

Reply:
I am investigating why using a **Server** value of `(local)` in the Tableau connection dialog FAILS with an error when using the Actian ODBC taco. The `connectionBuilder.js` looks like it built to handle this, but I haven't yet gotten it to work. Still looking into it.

Reply:
Any idea if the older taco(s) still work? Wondering if we are looking at a change in behavior in Tableau or our Tableau Connector?

Reply:
I opened Jira issue [II-10697](https://actian.atlassian.net/browse/II-10697) to explore the continued ability to use a **Server** value of `(local)` for connecting to a locally installed DMBS.

Reply:
After additional investigation and discussion, decided for now not to update the ODBC connector in regard to using a Server value of `(local)`. See [II-10697](https://actian.atlassian.net/browse/II-10697) for details.  cc: @clach04